src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -115,7 +115,8 @@ private Node setMouseClickForNode(Node node, String milestoneName) {
     }
 
     private void handleMouseClick(String milestoneName) {
-        inputField.setText(milestoneName);
+        inputField.setDisable(true);
```

Comment:
Yes, this has been discussed with @damithc. This was decided mainly due to 2 reasons:
- consistent with label picker
- the best matching milestones should already contain the milestone that the user wishes to pick